src/styles/style.css

```diff
@@ -1587,3 +1587,20 @@ label > .toggl-button.outlook {
 .toggl-button.dropbox-paper {
   vertical-align: middle;
 }
+
+/****** WORKFLOWY ******/
+.workflowy-toggl-wrapper {
+  display: flex !important;
+}
+
+.toggl-button.workflowy {
```

Comment:
This is a global problem, but in the meantime I think we'll need to add `:not(.toggl-button-edit-form-button)` to stop some of these styles leaking to the popup that appears after you click a button.

Reply:
It would also be a bonus if the styles in the menu could be tweaked so that it fits more naturally in the Workflowy UI. At the moment, the text and icon are noticeably bigger.

![Screen Shot 2019-03-25 at 16 49 03](https://user-images.githubusercontent.com/6432028/54938302-01d72180-4f1e-11e9-9534-af10fb8bea49.png)